config/webpack/webpack.config.base.js

```diff
@@ -134,10 +134,6 @@ module.exports = (env) => {
         'core-js/library/fn/array/includes': path.resolve(NODE, 'core-js-pure/features/array/includes'),
       },
       extensions: ['.js', '.css'],
-      modules: [
-        SOURCE,
-        NODE,
-      ],
```

Comment:
I was able run the app locally without having to make this change. I don't think this is necessary.

Reply:
Interesting. I was not...

Reply:
Hmmm ... let me try that.

Reply:
I cannot reproduce, neither with npm v7 nor npm v8.

Reply:
not crashing anymore, but warnings are still issued that are resolved with this change